ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -170,6 +170,16 @@ gcp_connection::object_exists(const std::string &object_key, bool &exists, size_
     }
 
     std::cerr << object_key + ": " + metadata.status().message() << std::endl;
+    return print_error_info(metadata.status());
+}
 
+int 
+gcp_connection::print_error_info(const google::cloud::Status status) const 
+{
+    std::cerr << status.message() << std::endl;
+    if (toErrno.find(status.code()) != toErrno.end())
+        return (toErrno.at(status.code()));
+    std::cerr << status.code() << std::endl;
```

Comment:
is there a point to printing out the http code here if you are returning the mapped system error code?

Reply:
I am only printing out the status code if a mapping is not found

Reply:
i guess this depends on what is most important to return if the function that uses `print_error_info` needs an OS error then maybe -1 would be better here just my thoughts